### DISCUSSION TITLE
issue met control node?

### DISCUSSION THREAD

**Discussion by HOUJ2** [Deltares/Ribasim#486] [Q&A] [answered]
Mijn collega (nu met vakantie) heeft voor haar vakantie een klein test modelletje opgezet in Qgis met een control node en kwam daarin iets geks tegen. Ik zet de opmerking door:

Je moet nu voor elke edge aangeven om wat voor edge het gaat (een flow of control edge). Dit hoort waarschijnlijk in de attribute table bij edge te kunnen, maar die kolom is afwezig. Dit geeft een foutmelding bij het runnen. Ik heb toen handmatig die kolom toegevoegd en aangegeven of het om een flow of control edge gaat, maar op de een of andere manier pakt die ‘control’ niet (als je het opslaat wordt ‘control’ automatisch weer teruggezet naar ‘flow’). Vervolgens krijg ik bij het runnen de foutmelding “Cannot connect a Control to a Pump”… Dat zal daaraan liggen. 

Enig idee waaraan dit zou kunnen liggen? Hopelijk hebben jullie er iets aan, anders mogelijk even wachten tot mijn collega het weer kan reproduceren.

**Comment by evetion** [comment]
Dankjewel voor het rapporteren! Zo te lezen hebben jullie een oude QGIS plugin versie, waar het edge_type attribuut nog niet in voorkomt. Bij het testen merkte ik echter dat ook in recente versies het edge_type niet correct gezet kan worden, ik werk aan een PR.

Ik geef een seintje als we een werkende versie online hebben staan.

**Reply by evetion** [reply, marked as answer]
De laatste release https://github.com/Deltares/Ribasim/releases/tag/v2023.08.0 heeft een QGIS zip die zou moeten werken.
